packages/helper-grammar-regex-collection/index.js

```diff
@@ -105,4 +110,5 @@ export {
   CSS_IMPORT,
   HTML_IMPORT,
   go,
+  HASKELL_IMPORT,
```

Comment:
Can you add some tests to verify that the regex is working as expected, please. It's easy and it shouldn't take much time [see](https://github.com/ousennin/browser-extension/blob/master/packages/helper-grammar-regex-collection/test.js#L168-L183)